repository-elasticsearch/src/main/java/org/kestra/repository/elasticsearch/ElasticSearchFlowRepository.java

```diff
@@ -105,7 +104,10 @@ private static String flowUid(Flow flow) {
     }
 
     public Flow create(Flow flow) throws ConstraintViolationException {
-        return this.save(flow);
```

Comment:
This is an emit event flow proposal. on create I register flow first, then I save on delete I trigger unschedule first. I am not sure about what is the best for update.